### ISSUE TITLE
Markdown linting: Fenced code blocks should have a language specified (MD040)

### ISSUE THREAD

**Issue by dshevtsov** [magento/devdocs#5252]
# Bug report

## Description

The Markdown linter reports about 1421 issues when the [rule 'MD040']( https://github.com/markdownlint/markdownlint/blob/master/docs/RULES.md#md040---fenced-code-blocks-should-have-a-language-specified) is enabled.

### Steps to reproduce

1. In the `_checks/styles/style-rules-prod` change `exclude_rule 'MD040'` to `rule 'MD040'`.
2. In your terminal, run `rake test:md`.

## Test your updates

1. Follow "Steps to reproduce" locally to get list of errors to fix.
2. Preview the HTML locally to ensure your changes did not break formatting in the resulting HTML (run `rake:preview`). For more information, refer to the [README](https://github.com/magento/devdocs#to-build-locally).

**Note**-->Do NOT use [special keywords](https://help.github.com/en/articles/closing-issues-using-keywords#about-issue-references) in your commit message.

## Expected result

All the reported issues are fixed.
The `rake test:md` command returns `No issues found`.

**Comment by jeff-matthews** [comment]
This rule flags indented code blocks as errors; not just fenced code blocks with no language specified. See the following example:

<img width="1031" alt="Screen Shot 2019-09-19 at 10 32 28 AM" src="https://user-images.githubusercontent.com/13662379/65258814-30971480-dac9-11e9-8488-3ddb657c0ead.png">

That's going to make it more difficult to enable this rule, especially when we have a separate issue for enforcing the fenced code block style (#5253).

**Comment by shrielenee** [comment]
@danidelcar This will affect OMS Docs significantly when/if this is merged to our repo. I think I brought this up to you recently when doing a review.

**Comment by jeff-matthews** [comment]
There are still a lot of errors to fix, but there are two that are in the MRG:

```
guides/v2.3/mrg/ce/InstantPurchase.md:35: MD040 Fenced code blocks should have a language specified
guides/v2.3/mrg/ce/InstantPurchase.md:43: MD040 Fenced code blocks should have a language specified
```

**Comment by jeff-matthews** [comment]
PR #5478 brings the error count down to ~`265`.

**Comment by jeff-matthews** [comment]
After #5495, #5497, and #5498 are merged, we can turn on the MD040 rule in production.
